### ISSUE TITLE
media-libs/vapoursynth-dfttest build issues

### ISSUE THREAD

**Issue by olifre** [4re/vapoursynth-portage#1]
Hi, 

apparently we had a very similar idea, I had also started packaging vapoursynth and some plugins: 
https://bugs.gentoo.org/show_bug.cgi?id=537092
but it seems your tree is much more advanced, thanks for it! Maybe it would be a better idea to try to get your more sophisticated ebuilds into portage? Feel free to replace my bug reports attached to 537092 with your versions of the ebuilds, if you like. 

However, to the problem at hand: 
vapoursynth-dfttest is now testing for fftw3f in configure and tries to resolve a symbol of libfftw3f_threads.so inside fftw3f.so, which fails. 
To fix it: 
https://537892.bugs.gentoo.org/attachment.cgi?id=396642
In short, the dependency should be: 
sci-libs/fftw:3.0[threads]
and you need
sed -i 's/DEPLIBS="fftw3f"/DEPLIBS="fftw3f_threads"/' configure
in src_configure before calling configure. 

It also seems "/lib/" is still hardcoded here instead of get-libdir. 
In my ebuilds, I used the approach of installing to the normal libdir, and then placing a symlink in /usr/$(get-libdir/vapoursynth, not sure if this is really a better approach. 

Cheers, 
Oliver

**Comment by 4re** [comment]
Hi there,

actually I already saw your ebuilds the day I asked for the overlay, nice work there, I took some ideas from them. Feel free to send me any patches if you se something wrong here and thanks for this one.

Right now I'm in the middle of checking ebuilds and adding non-live packages to them, I will try to finish it tomorrow to push your patch there.

**Comment by 4re** [comment]
Nevermind, I pushed it now just with this changes via c82217f824052a14cdaec7874b028e42bfba09a2 .

**Comment by olifre** [comment]
Thanks! 
I will switch from my ebuilds to using your overlay and test a bit in the near future, it really is better to channel these efforts to a common place. 
I have also added a comment to https://bugs.gentoo.org/show_bug.cgi?id=537092 so users looking there will also become aware of your overlay - I could never have found the time to package all these plugins, thanks a lot for your hard work! 
Cheers, Oliver
